instrumentation/jsonrpc4j-1.3/library/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+val jacksonVersion = "2.13.3"
```

Comment:
You can remove this. Jackson version is set by dependency management so you could just use `library("com.fasterxml.jackson.core:jackson-databind")`